docs/conf/na/2017/news/schedule-social.rst

```diff
@@ -0,0 +1,61 @@
+:template: 2017/na-content.html
+
+.. post:: May 1, 2017
+   :tags: 2017, portland, schedule, social
+
+Announcing Full Schedule and Social Events
+==========================================
+
+Our Portland conference is just two weeks away and we hope you're getting excited!
+
+We are officially sold-out but you can sign up for the `waiting list <https://ti.to/writethedocs/write-the-docs-na-2017>`_ in case we have any last-minute cancellations.
```

Comment:
Should probably drop the waiting list bit, as when this goes out folks will probably be too late, and I don't want to deal with more waiting list people.